src/pointer-analysis/dereference.cpp

```diff
@@ -46,6 +44,9 @@ exprt dereferencet::operator()(const exprt &pointer)
     type);
 }
 
+/// \return expression equivalent to `byte_extract(object, offset)` and of type
+///   `type`. The expression is potentially simplified so that for instance in
+///   the case of an array, `object[offset]` can be returned.
```

Comment:
In a number of places in this file, you have not included \param tags, or \return tags. We seem to be working with a policy of always including them, even where they don't add a huge amount of additional info.

Reply:
I don't see anything in the coding standard which says we have to do so. In lot of cases `\param` tags add no information at all.

Reply:
I could be mistaken :-) (apologies if so) - It just seemed like a general unwritten policy. That's not my blocking point though so I won't push too hard.